paraview/xmls/RipsPersistenceDiagram.xml

```diff
@@ -0,0 +1,90 @@
+<ServerManagerConfiguration>
+  <ProxyGroup name="filters">
+    <SourceProxy name="ttkRipsPersistenceDiagram" class="ttkRipsPersistenceDiagram" label="TTK RipsPersistenceDiagram">
+      <Documentation long_help="RipsPersistenceDiagram Long" short_help="RipsPersistenceDiagram Short">
+        TTK plugin for the computation of the persistence diagram of Rips complexes.
+
+        This plugin computes the persistence diagram of the Rips complex of the
+        input point cloud or distance matrix. The X-coordinate of each pair
+        corresponds to its birth, while its smallest and highest Y-coordinates
+        correspond to its birth and death respectively.
+
+        In practice, the diagram is represented by a vtkUnstructuredGrid. Each
+        vertex of this mesh represent a critical point of the input data. It is
+        associated with point data (vertexId, critical type). Each vertical edge
+        of this mesh represent a persistence pair. It is associated with cell data
+        (persistence of the pair, critical index of the extremum of the pair).
+        The diagonal of the diagram can be filtered out by considering its
+        PairIdentifier value (set at -1).
+      </Documentation>
+
+      <!-- INPUT DATA OBJECTS -->
+      <InputProperty name="Input" command="SetInputConnection">
+        <ProxyGroupDomain name="groups">
+          <Group name="sources" />
+          <Group name="filters" />
+        </ProxyGroupDomain>
+        <DataTypeDomain name="input_type">
+          <DataType value="vtkTable" />
+        </DataTypeDomain>
+        <InputArrayDomain name="input_scalars" number_of_components="1" />
```

Comment:
I don't believe this line is necessary (it basically says this filter expects a scalar field, which is well defined for `vtkDataSet` objects, not for `vtkTable` objects I believe).